src/gui/EffectSelectDialog.cpp

```diff
@@ -206,8 +206,8 @@ void EffectSelectDialog::rowChanged( const QModelIndex & _idx,
         {
             QLabel *label = new QLabel(m_descriptionWidget);
             QString labelText = "<p><b>" + tr("Name") + ":</b> " + QString::fromUtf8(descriptor.displayName) + "</p>";
-            labelText += "<p><b>" + tr("Description") + ":</b> " + QString::fromUtf8(descriptor.description) + "</p>";
-            labelText += "<p><b>" + tr("Author") + ":</b> " + QString::fromUtf8(descriptor.author) + "</p>";
+						labelText += "<p><b>" + tr("Description") + ":</b> " + qApp->translate( "pluginBrowser", descriptor.description ) + "</p>";
+						labelText += "<p><b>" + tr("Author") + ":</b> " + QString::fromUtf8(descriptor.author) + "</p>";
```

Comment:
Hi Tres,

You are right.
It is somewhat difficult to follow the formatting in LMMS since some 
sources uses tabs, some don't.
Looks like this one does not :-\

midi-pascal

On 15-05-21 01:23 PM, Tres Finocchiaro wrote:
>
> In src/gui/EffectSelectDialog.cpp 
> <https://github.com/LMMS/lmms/pull/2065#discussion_r30824701>:
>
> > @@ -206,8 +206,8 @@ void EffectSelectDialog::rowChanged( const QModelIndex & _idx,
> >          {
> >              QLabel *label = new QLabel(m_descriptionWidget);
> >              QString labelText = "<p><b>" + tr("Name") + ":</b> " + QString::fromUtf8(descriptor.displayName) + "</p>";
> > -            labelText += "<p><b>" + tr("Description") + ":</b> " + QString::fromUtf8(descriptor.description) + "</p>";
> > -            labelText += "<p><b>" + tr("Author") + ":</b> " + QString::fromUtf8(descriptor.author) + "</p>";
> > +						labelText += "<p><b>" + tr("Description") + ":</b> " + qApp->translate( "pluginBrowser", descriptor.description ) + "</p>";
> > +						labelText += "<p><b>" + tr("Author") + ":</b> " + QString::fromUtf8(descriptor.author) + "</p>";
>
> I think we have some whitespace formatting issues here...
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/LMMS/lmms/pull/2065/files#r30824701>.
>